tox.ini

```diff
@@ -343,7 +343,8 @@ commands =
 
 [testenv:test-release]
 basepython = python3
-skip_install = true
+# Set skip_install false because we need to install the cli for sphinx.
```

Comment:
Nit:
```suggestion
# Set skip_install false because we need to install the aws-encryption-sdk-cli for sphinx.
```

Reply:
Fair enough. My assumption was that it is obvious enough.